http/protocolversioning.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>Test HTTP's versioning scheme</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id=log></div>
+<script>
+[
+  "HTTP/0.7 280",
+  "HTTP/1.0 280",
```

Comment:
syntactically, a reason-phrase is required; you might want to include them so you don't get false negatives.